cmd2/cmd2.py

```diff
@@ -3185,232 +3125,7 @@ def restore(self):
                 setattr(self.obj, attrib, getattr(self, attrib))
 
 
-class OutputTrap(object):
```

Comment:
Thanks for moving all of the transcript testing stuff to its own file.  **cmd2.py** is now feeling a little more focused